examples/02_model_collaborative_filtering/multi_vae_deep_dive.ipynb

```diff
@@ -0,0 +1,1829 @@
+{
```

Comment:
<p>One thing that is missing in my view is a discussion on the key contributions and differences of Multi-VAE: the multinomial likelihood and the regularization using bayesian inference.</p><p> </p><p>They use multinomial likelihood which is good for modeling implicit feedback, and behaves better than logistic or gaussian likelihoods, which are typically more popular in the literature. By using multinomial likelihood, they are treating the problem as multi-class classification. As the authors discuss in the paper: "this likelihood rewards the model for putting probability mass on the non-zero entries in xu . But the model has a limited budget of probability mass, since π(zu) must sum to 1; the items must compete for this limited budget. The model should therefore assign more probability mass to items that are more likely to be clicked"</p><p> </p><p>They use bayesian inference for parameter estimation. The variational inference model q(zu|xu) approximates the intractable posterior distribution p(zu|xu). The authors interpret the standard VAE objective as a regularization parameter to better optimize the model.&nbsp;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>